specs/StoragePickers/PickFileResult.md

```diff
@@ -0,0 +1,88 @@
+PickFileResult Class
+===
+
+# Background
+
+Namespace: [Microsoft.Windows.Storage.Pickers](./Microsoft.Windows.Storage.Pickers.md)
+
+Represents the result of a file picking operation. This is a lightweight class that contains a 
+string attribute representing the file path. Its class object can be converted to a 
+`Windows.Storage.StorageFile` object.
+
+# API Pages
+
+## Constructor
+
+### Attributes
+
+| **Attribute** | **Type** | **Description** |
+|---------------|----------|-----------------|
+| `Path`        | `string` | Gets the path of the picked file. |
+
+### Examples
+C#
+
+```csharp
+using Microsoft.Windows.Storage.Pickers;
+
+var result = new PickFileResult("C:\\example\\file.txt");
```

Comment:
PickFileResult should not and can not be constructed from path, the only way to get PickFileResult is from pickers